<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusWidget</class>
 <widget class="QWidget" name="StatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="statusLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lbStatusIcon">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbStatusText">
     <property name="text">
      <string>STATUS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbSerialPort">
     <property name="text">
      <string>PORT_NAME</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbSeialPortInfo">
     <property name="text">
      <string>PORT_INFO</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbRtsStatus">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="serial_monitor.qrc">:/serialmonitor/icons/status_red.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbRtsText">
     <property name="text">
      <string>RTS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbDtsStatus">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="serial_monitor.qrc">:/serialmonitor/icons/status_red.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lnDtsText">
     <property name="text">
      <string>DTS</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="serial_monitor.qrc"/>
 </resources>
 <connections/>
</ui>
